integtest/spec/all_books_broken_link_detection_spec.rb

```diff
@@ -156,6 +156,12 @@
       include_examples 'there are broken links in kibana',
                        'en/fleet/master/not-a-fleet-page.html'
     end
+    describe 'when there is a broken APM Get Started link' do
+      include_context 'there is a APM Get Started link', true,
```

Comment:
```suggestion
      include_context 'there is a kibana link', true,
```

This actually refers to a block of setup code (see line 28). 

```
03:23:51 An error occurred while loading ./spec/all_books_broken_link_detection_spec.rb.
03:23:51 Failure/Error:
03:23:51   include_context 'there is a APM Get Started link', true,
03:23:51                   '${APM_GET_STARTED}not-an-apm-page.html', true
03:23:51 
03:23:51 ArgumentError:
03:23:51   Could not find shared context "there is a APM Get Started link"
```